File: AdminService/src/main/java/com/ecommercial/shopping/adminservice/global/exception/MyException.java
package com.ecommercial.shopping.adminservice.global.exception;

import lombok.AllArgsConstructor;
import lombok.Getter;
import org.springframework.http.HttpStatus;

@Getter
public class MyException extends RuntimeException{
    private final HttpStatus exceptionCode;
    private final String exceptionMessage;

    public MyException(HttpStatus exceptionCode, String exceptionMessage) {
        super(exceptionMessage);
        this.exceptionCode = exceptionCode;
        this.exceptionMessage = exceptionMessage;
    }

}

File: UserService/src/main/kotlin/com/ecommercial/shopping/userservice/global/exception/MyException.kt
package com.ecommercial.shopping.userservice.global.exception

import lombok.AllArgsConstructor
import lombok.NoArgsConstructor
import org.springframework.http.HttpStatus

@AllArgsConstructor
@NoArgsConstructor
class MyException(
    val httpSecurity: HttpStatus,
    val exceptionMessage: String
) : RuntimeException(){
}